speedtest/assets/dashboards/speedtest.json

```diff
@@ -0,0 +1,325 @@
+{
+    "author_info": {
+        "author_name": "Datadog"
+    },
+    "board_title": "Speedtest",
+    "description": "",
+    "template_variables": [
+        {
+            "default": "*",
+            "name": "env",
+            "prefix": "env"
+        },
+        {
+            "default": "*",
+            "name": "isp",
+            "prefix": "isp"
+        },
+        {
+            "default": "*",
+            "name": "host",
+            "prefix": "host"
+        }
+    ],
+    "widgets": [
+        {
+            "height": 13,
+            "legend": false,
+            "legend_size": "0",
+            "tile_def": {
+                "requests": [
+                    {
+                        "q": "avg:speedtest.ping.jitter{*,*,*} by {isp}",
+                        "style": {
+                            "palette": "dog_classic",
+                            "type": "solid",
+                            "width": "normal"
+                        },
+                        "type": "line"
+                    }
+                ],
+                "viz": "timeseries",
+                "yaxis": {
+                    "includeZero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "title": true,
+            "title_align": "left",
+            "title_size": 16,
+            "title_text": "Ping Jitter (ms)",
+            "type": "timeseries",
+            "width": 47,
+            "x": 97,
+            "y": 21
+        },
+        {
+            "height": 13,
+            "legend": false,
+            "legend_size": "0",
+            "tile_def": {
+                "requests": [
+                    {
+                        "q": "avg:speedtest.ping.latency{*,*,*} by {isp}",
+                        "style": {
+                            "palette": "dog_classic",
+                            "type": "solid",
+                            "width": "normal"
+                        },
+                        "type": "line"
+                    }
+                ],
+                "viz": "timeseries",
+                "yaxis": {
+                    "includeZero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "title": true,
+            "title_align": "left",
+            "title_size": 16,
+            "title_text": "Ping Latency (ms)",
```

Comment:
Should the titles follow our guidelines (note widget, gray background, 18px, pointer down)? 

https://datadoghq.dev/integrations-core/guidelines/dashboards/#best-practices

Reply:
Good comment - updated accordingly.